src/SIS_dyn_cgrid.F90

```diff
@@ -2083,10 +2124,13 @@ subroutine SIS_C_dyn_read_alt_restarts(CS, G, US, Ice_restart, restart_dir)
                      (US%kg_m3_to_R_restart * US%m_to_Z_restart * US%m_to_L_restart**2)
     do J=G%jsc-1,G%jec ; do I=G%isc-1,G%iec
       CS%str_s(I,J) = stress_rescale * CS%str_s(I,J)
+      if (abs(CS%str_s(I,J)) < CS%str_underflow) CS%str_s(I,J) = 0.0
```

Comment:
I omitted that one because it only occurs at initialization from an older-format restart file that will no longer be written for versions of the code subsequent to https://github.com/NOAA-GFDL/SIS2/pull/175.